### PR TITLE
Add Spanish translations

### PR DESCRIPTION
**Pull request by MinThaMie**

### CODE REVIEW

File: translations/es.yaml [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+  my-roles: 'Mis Roles'
+  no-roles: 'No has añadido ningún rol, echa un vistazo abajo para ver en qué puedes contribuir.'
+  bingo-title: 'Diferentes formas de contribuir'
+  cancel: 'Volveré más tarde'
+  add: 'Añadir a mis roles'
+  remove: 'Eliminar de mis roles'
+  roles:
+    ambassador:
+      title: 'Embajador'
+      description: 'Tomé el entrenamiento de Embajador para la Inclusión y promuevo la diversidad, equidad e inclusión.'
```
Comment (yagodela) [inline translations/es.yaml]:
```suggestion
      description: 'Hice el entrenamiento de Embajador para la Inclusión y promuevo la diversidad, equidad e inclusión.'
```

File: translations/es.yaml [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+    inspector:
+      title: 'Inspector'
+      description: 'Identifico y abordo problemas de accesibilidad en nuestros productos y lugar de trabajo.'
+      explanation: ''
+    wordsmith:
+      title: 'Experto en Palabras'
+      description: 'Me aseguro de que nuestro mensaje sea inclusivo.'
+      explanation: ''
+    sharer:
+      title: 'Compartidor'
```
Comment (yagodela) [inline translations/es.yaml]:
```suggestion
      title: 'Divulgador'
```

File: translations/es.yaml [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+      title: 'Viajero del Mundo'
+      description: 'Interactúo con colegas internacionales y tengo curiosidad por su perspectiva.'
+      explanation: ''
+glossary:
+  subtitle: 'Consulta aquí'
+  title: 'Vocabulario DEI'
+  welcome: '¿No estás seguro de lo que significan algunos términos? ¡No hay problema! En esta vista general encontrarás muchos términos que puedes encontrar dentro de tu organización.'
+  placeholder: 'Escribe aquí para filtrar el glosario'
+  search: 'Buscar en el glosario'
+  go-to: 'Go to'
```
Comment (yagodela) [inline translations/es.yaml]:
```suggestion
  go-to: 'ir a'
```

**Review by yagodela — COMMENTED** [review]
Hey @MinThaMie 👋, I reviewed the document and is fine 👌, I just suggest a couple of changes.

**Comment by MinThaMie** [comment]
Thank you very much :)

**Pull request merged**